agent_app_sample_code/tests/test_file_loading.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import pyspark
+import pandas as pd
+
+from agent_app_sample_code.utils.file_loading import load_files_to_df
+
+@pytest.fixture(scope="module")
+def spark():
+    return (
+        pyspark.sql.SparkSession.builder
+        .master("local[1]")
+        # Uncomment the following line for testing on Apple silicon locally
```

Comment:
can we create some control flag such that we can run `pytest` locally (on apple silicon) and get this to work and then CI can have the flag the other way.

Reply:
Good call - actually, looks like CI seems to work with this stuff enabled by default too? So will just remove the comment